src/couch/test/eunit/couchdb_auth_tests.erl

```diff
@@ -21,24 +21,36 @@ setup(PortType) ->
     Addr = config:get("httpd", "bind_address", "127.0.0.1"),
     lists:concat(["http://", Addr, ":", port(PortType), "/_session"]).
 
+setup_require_valid_user(PortType) ->
+    ok = config:set("couchdb", "require_valid_user", "true", _Persist=false),
```

Comment:
"couchdb" -> "chttpd"  ?

Reply:
Whoops, good catch! Fixed now (and confirmed that the test actually fails without the corresponding patch to `chttpd_auth`)